app/controllers/councillor_contributions_controller.rb

```diff
@@ -23,6 +23,7 @@ def create
     end
 
     if @councillor_contribution.save
+      AdminNoticeMailer.notice_for_councillor_contribution.deliver_later
```

Comment:
The other way to do this is with a callback on 'create' for the contribution—but I *much* prefer this approach of directly making the call to email where we want it sent, because it's much easier to follow and test because it's more predictable ✨